### DISCUSSION TITLE
Unable to show many to many displate template name

### DISCUSSION THREAD

**Discussion by cercec** [directus/directus#12883] [Archive]
<img width="927" alt="image" src="https://user-images.githubusercontent.com/60812978/164270052-3b95c35a-f76d-4b6a-9428-5a17e5b8a7f3.png">

When I want to set a many to many, only the id show, and not the name of the referring data.

**Comment by rijkvanzanten** [comment]
That can be configured through the `Display Template` option in the Interface's options:

<img width="680" alt="CleanShot 2022-04-20 at 11 52 19@2x" src="https://user-images.githubusercontent.com/9141017/164271924-07e530e9-aa83-4202-a853-719488d84e9f.png">
